server/integration_tests/test_data/explore_strict_default_place/howtoearnmoneyonlinewithoutinvestment/chart_config.json

```diff
@@ -1,24 +1,16 @@
 {
+  "blocked": true,
```

Comment:
why is blocked true in this case?

Reply:
ah, accidentally passed that to the abort message, fixed